xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -49,6 +49,18 @@ bool CPVRChannelGroupInternal::Load(
 {
   if (CPVRChannelGroup::Load(channels))
   {
+    for (const auto& groupMember : m_members)
+    {
+      const std::shared_ptr<CPVRChannel> channel = groupMember.second->Channel();
+
+      // create the EPG for the channel
+      if (channel->CreateEPG())
```

Comment:
Can channel be null here?

Reply:
No, it must not. if it is, we would face a bug which must not be worked around by nullptr check. :-).

Reply:
Cool

Reply:
how about logging an error and returning false in case of a nullptr? better than crashing imo. and i would not have to provide a bt when i face this bug :)

Reply:
No, it has to crash. Hiding serious programming errors you can't recover from is something I'm strictly against, sorry.